docs/android.md

```diff
@@ -109,6 +142,37 @@ those sections before configuring and building OpenDDS.
 PATH=$PATH:$TOOLCHAIN/bin make # Pass -j/--jobs with an appropriate value or this'll take a while...
 ```
 
+**Windows Users:** The command is similar with these exceptions:
+
+- `--host-tools` with the location of the OpenDDS host tools that were built
+  using Visual Studio must be passed to `configure`.
+
+- In addition to the Android toolchain, you will also need MinGW utilities in
+  your `%PATH` and the command prompt environment for Visual Studio. The
+  default installation location of the MinGW utilities is
+  `C:\MinGW\msys\1.0\bin`. The Visual Studio command prompt is the same as when
+  configuring the host tools. Well actually it can't be the exact same still
+  running environment or else the `ACE_ROOT` and `TAO_ROOT` environment
+  variables will mess up the build for Android, so start a new one if you
+  haven't already.
+
+In the end the commands will look like this:
+
+```Batch
+configure --doc-group --target=android --macros=ANDROID_ABI=armeabi-v7a --host-tools=%HOST_DDS%
+call "C:\Program Files (x86)\Microsoft Visual Studio\Version\Offering\Common7\Tools\VsDevCmd.bat"
+set PATH=%TOOLCHAIN%/bin;%PATH%
+set PATH=C:\MinGW\msys\1.0\bin;%PATH%
+make
+REM Pass -j/--jobs with an appropriate value or this'll take a while...
+```
+
+You can get rid of the second line if you're already in a Visual Studio
+"Developer Command Prompt" or adjust it if your Visual Studio is different
+version or configuration. See
+[here](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/how-to-set-environment-variables-for-the-visual-studio-command-line)
```

Comment:
This is in the C# reference guide, is it relevant?  I've never used `VsDevCmd`.  The links that VS puts on the start menu use something like `...\VC\Auxiliary\Build\vcvarsx86_amd64.bat`

Reply:
I didn't even notice that. I think it's all the same for what we're doing because it worked for me.

Reply:
Let's avoid having additional details here compared to INSTALL and configure which just say "Visual Studio Command Prompt" should be used.  "Developer Command Prompt" is actually a different thing.  It defaults to 32-bit... probably a .NET thing?

Reply:
I still don't really understand, but sure.